new/LIT7041LordsPrayerAmh.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7041LordsPrayerAmh" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">የጌታ፡ ጸሎት፡</title>
+                <title xml:lang="am" type="normalized" corresp="#t1">Yagetā ṣalot</title>
+                <title xml:lang="en" corresp="#t1">Lord's prayer</title>
+                <title xml:lang="am" xml:id="t2">አባታችን፡ ሆይ፡ በሰማያት፡ የምትኖር፡</title>
+                <title xml:lang="am" type="normalized" corresp="#t2"></title>
+                <title xml:lang="en" corresp="#t2">Our Father, who dwells in heaven</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Matthew 6:9-15 in Amharic.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="ChristianContent"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-07-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7041LordsPrayerAmh" passive="BLorient7946"/>
```

Comment:
There is the Geez text which probably has to be interlinked this record, but the main question for me is:
do we creat special records for Amharic "doublets"/"translations" of Geez texts? Or we record the existance of another translation in the sam record? (cp. LIT2168PaterN) Or we decide in each individual case? This possible should be discussed (when Zhenia is back?).

Reply:
Yes, we can discuss later. However, we do have already a good number of Old Testament books in Amharic and a discussion about this principal question in https://github.com/BetaMasaheft/Documentation/issues/2476. You can open this issue again. However, I think, it makes sense to have several IDs for each language version. A translation is a significant change and a new version whatsoever.

Reply:
I remember it was said that some biblical books and important texts may be exceptions, but otherwise we kept it strickt.